crates/router/src/connector/stripe/transformers.rs

```diff
@@ -3204,7 +3204,29 @@ pub struct WebhookPaymentMethodDetails {
 }
 
 #[derive(Debug, Deserialize)]
-pub struct WebhookEventObjectData {
+#[serde(untagged)]
+pub enum WebhookEventObjectData {
+    WebhookEventObjectDataWithMetaData(WebhookEventObjectDataWithMetaData),
+    WebhookEventObjectDataWithoutMetaData(WebhookEventObjectDataWithoutMetaData),
+}
+
+#[derive(Debug, Deserialize)]
+pub struct WebhookEventObjectDataWithMetaData {
```

Comment:
Instead of having this new struct, let's make metadata->order_id as option field.

Reply:
done👍